slsDetectorGui/include/qTabSettings.h

```diff
@@ -52,9 +58,21 @@ class qTabSettings : public QWidget, private Ui::TabSettingsObject {
         G2_LOWCAP_LOWGAIN,
         G4_HIGHGAIN,
         G4_LOWGAIN,
+        GAIN0,
         UNDEFINED,
         UNINITIALIZED,
         NUMSETTINGS
     };
+
+    enum {
```

Comment:
Is it possible to reuse the enums from sls_detector_defs.h instead of defining new ones?

Reply:
I tried this, but then undefined and uninitialized has enum values 200.. and i wanted this to get the values from the drop down